census/migrations/20161027095620-add-entry-aboutyouanswers.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    return queryInterface.addColumn('entry', 'aboutYouAnswers',
+      {
+        type: Sequelize.JSONB,
+        allowNull: true,
```

Comment:
I think it's okay. In reality it won't be for new entries, but we have a lot of legacy entries that don't include this data.